packages/db/src/resources/types.ts

```diff
@@ -52,7 +52,7 @@ export type Collections = {
   contractInstances: {
     resource: DataModel.ContractInstance;
     input: DataModel.ContractInstanceInput;
-    idFields: ["address", "network"];
+    idFields: ["contract", "address", "creation"];
```

Comment:
This is fine but we really should consider/plan a migration system post-haste. What if you hadn't realized the need for this until after we released DB into the wild?

Reply:
Yeah, I know :o 

See below... I already have another change that I foresee here, for after release :(